Src/java/cql-to-elm/src/main/java/org/cqframework/cql/cql2elm/LibraryBuilder.java

```diff
@@ -2064,6 +2096,139 @@ else if (currentType instanceof ListType && listTraversal) {
         return null;
     }
 
+    // TODO: Support case-insensitive models
+    public List<MatchIdentifier> resolveProperties(DataType sourceType, String identifier, boolean mustResolve) {
```

Comment:
As I suggested in my last review, I really think IdentifierResolution is a better name for the concept here, and more consistent with other classes used in the translator and builder. Can we rename this class please?